app.js

```diff
@@ -7,6 +7,13 @@ const govukTemplate = require('hof-govuk-template');
 const hoganExpressStrict = require('hogan-express-strict');
 const expressPartialTemplates = require('express-partial-templates');
 const _ = require('lodash');
+const apiCall = require('./lib/api-call');
+const config = require('./config');
+const baseUrl = config.apiBaseUrl;
+const surveyID = config.surveyID;
+const responses = config.endPointResponses;
+const apiToken = config.apiToken;
+const apiTokenSecret = config.apiTokenSecret;
 
```

Comment:
I had the above const in the api-call file itself but I found it tricky to tests so I've moved them